src/fastdiff.js

```diff
@@ -101,12 +101,13 @@ export default function fastDiff( a, b, cmp, atomicChanges = false ) {
 	};
 
 	// Transform text or any iterable into arrays for easier, consistent processing.
+	// Array.from was used here but it generated incorrect results for multi-byte unicode sequences.
 	if ( !Array.isArray( a ) ) {
-		a = Array.from( a );
+		a = Array.prototype.map.call( a, c => c );
```

Comment:
Can't be `Array.from( a, c => c );` [according to docs](https://developer.mozilla.org/pl/docs/Web/JavaScript/Referencje/Obiekty/Array/from)?

Reply:
No, `Array#from` works differently for multi-byte characters.